src/pages/AddItemPage/AddItemPage.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import {
+  Button,
+  Container,
+  FlexContainer,
+  SectionTitle,
+} from "../../styles/CommonStyles";
+import styled from "styled-components";
+import InputItem from "../../components/UI/InputItem";
+import TagInput from "../../components/UI/TagInput";
+import ImageUpload from "../../components/UI/ImageUpload";
+
+const TitleSection = styled(FlexContainer)`
+  margin-bottom: 16px;
+`;
+
+const InputSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+
+  @media ${({ theme }) => theme.mediaQuery.tablet} {
+    gap: 24px;
+  }
+`;
+
+function AddItemPage() {
+  const [name, setName] = useState("");
+  const [description, setDescription] = useState("");
+  const [price, setPrice] = useState("");
+  const [tags, setTags] = useState([]);
```

Comment:
배열의 경우에도 안에 어떤 값이 올 지 적어주심 좋아요!
```suggestion
  const [tags, setTags] = useState<Array<string>>([]);
```